src/openforms/analytics_tools/contrib/expoints/csp_headers.json

```diff
@@ -1,7 +1,7 @@
 [
     {
         "directive" : "connect-src",
-        "value": "https://*.expoints.nl"
+        "value": "'self' https://sentry.maykinmedia.nl https://*.expoints.nl"
```

Comment:
@sergei-maertens I'm not sure how else to do this, I think previously, if no analytics tool that added `connect-src` (or another specific directive) was used, it would automatically include these, but once you set that directive, you lose these defaults